modules/dataquery/jsx/welcome.tsx

```diff
@@ -0,0 +1,1097 @@
+ * @param {function} props.onContinue - Callback when the "Continue" button is called in the welcome message
+ * @param {boolean} props.queryAdmin - True if the current user can pin study queries
+ * @param {function} props.reloadQueries - Reload the list of queries from the server
+ * @param {function} props.loadQuery - Load a query to replace the active query
+ * @param {function} props.starQuery - Function that will star a query
+ * @param {function} props.unstarQuery - Function that will unstar a query
+ * @param {function} props.shareQuery - Function that will share a query
+ * @param {function} props.unshareQuery - Function that will unshare a query
+ * @param {function} props.getModuleFields - Retrieve a module's fields from the backend and populate them into fulldictionary
+ * @param {object} props.mapModuleName - Function to map the backend module name to a user friendly name
+ * @param {object} props.mapCategoryName - Function to map the backend category name to a user friendly name
+ * @param {object} props.fulldictionary - The dictionary of all modules that have been loaded
+ * @returns {React.ReactElement} - The Welcome tab element
+ */
+function Welcome(props: {
+    recentQueries: FlattenedQuery[]
+    sharedQueries: FlattenedQuery[],
+    topQueries: FlattenedQuery[],
+
+    queryAdmin: boolean,
+
+    onContinue: () => void,
+    reloadQueries: () => void,
+    shareQuery: (queryID: number) => void,
+    unshareQuery: (queryID: number) => void,
+    starQuery: (queryID: number) => void,
+    unstarQuery: (queryID: number) => void,
+    getModuleFields: (module: string) => void,
+    loadQuery: (fields: APIQueryField[], filters: QueryGroup|null) => void,
+    mapModuleName: (module: string) => string,
+    mapCategoryName: (module: string, category: string) => string,
+    fulldictionary:FullDictionary,
+}) {
+    const panels: {
+        title: string,
+        content: React.ReactElement,
+        alwaysOpen: boolean,
+        defaultOpen: boolean,
+    }[] = [];
+    if (props.topQueries.length > 0) {
+        panels.push({
+            title: 'Study Queries',
+            content: (
+                <div>
+                    <QueryList
+                        queries={props.topQueries}
+                        loadQuery={props.loadQuery}
+                        defaultCollapsed={true}
+
+                        getModuleFields={props.getModuleFields}
+                        mapModuleName={props.mapModuleName}
+                        mapCategoryName={props.mapCategoryName}
+                        fulldictionary={props.fulldictionary}
+
+                        queryAdmin={props.queryAdmin}
+                        reloadQueries={props.reloadQueries}
+                    />
+                </div>
+                ),
+            alwaysOpen: false,
+            defaultOpen: true,
+        });
+    }
+    panels.push({
+            title: 'Introduction',
```

Comment:
maybe this can be 'Instructions' instead of Introduction to make the order of panels feel right